README.md

```diff
@@ -157,6 +157,106 @@ Gets the permission information of a record
 
 ---
 
+# Append data to Linnia
+
+In order to append data to Linnia your web3 instance need to be able to sign with the private key of the owner of the data.
+
+## linnia.addRecord
+
+```javascript
+linnia.addRecord(dataHash, metadata, dataUri, ethParams);
+```
+
+Add record to Linnia
+
+### Parameters
+
+1. `String` - The data hash. Hash of the plain text data + metadata
+2. `String|Object` - The metadata of the record. [Click here to read more about the metadata](https://github.com/ConsenSys/linnia-resources/blob/master/METADATA.md)
+3. `String` - The dataUri, link to the data (eg. the IPFS hash)
+4. `Object` - The ethParams ethereum account params. 
+
+### Returns
+
+`Promise<Record>` - A promise when resolved returns the record object that was stored.
+
+### Append data using Linnia Js in the browser
+
+ If you are using a Linnia Js in the browser you can create the Linnia instance with the web3 object from Metamask and append a record the following way:
+
+### Example
+
+```javascript
+const Linnia = require("@linniaprotocol/linnia-js");
+const linnia = new Linnia(web3);
+
+const dataHash = "0xcc85fc3d763b9a1d83e4386b37b4b0f3daf9881638ba8b7db0c501c417acb689";
```

Comment:
We should probably show how the dataHash is produced too if there's nowhere else in these documents that does it. Maybe start from stringified "{"foo": "bar"}" json?

Reply:
I created a card in Jira to create a sample code to explain how to add to IPFS, generate the keys, encrypt and the whole flow in order to add a file to Linnia using Linnia JS. Check it here:
https://landstream.atlassian.net/browse/LP-520